Fix T970871 and remove encodeHtml from docs (#2180)
DevExpress/devextreme-documentation: concepts/Common/Security Considerations/20 HTML Encoding/20 Encoding in Templates.md

> Angular, Vue, and React always encode values interpolated in templates. With other frameworks and libraries, use one of the sanitizing tools as follows:

Reviewer: "one of the sanitizing tools" - we didn't mention any sanitizing tools.

```suggestion
Angular, Vue, and React always encode values interpolated in templates. With other frameworks and libraries, use a third-party sanitizing tool as follows:
```

        content: "Tab content" 
    }];

When you insert unencoded content, it can open your application to XSS attacks:

![DevExtreme Tabs with disabled HTML encoding](/images/UiWidgets/tabs-itemTemplate-unsafe-html-wo-codepen.png)

The encoded content is interpreted and displayed as text:

> 

Reviewer: Let's rename the articles to avoid -ing:

Encoding User Input -> Encode User Input
Encoding in Templates -> Encode Template Data
